cmd/kubeadm/app/phases/addons/dns/manifests.go

```diff
@@ -213,6 +213,42 @@ spec:
     k8s-app: kube-dns
 `
 
+	// CoreDNSService is the coredns Service manifest
+	CoreDNSService = `
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    k8s-app: kube-dns
+    kubernetes.io/cluster-service: "true"
+    kubernetes.io/name: "CoreDNS"
```

Comment:
i'd prefer if you add a new template variable called `{{ .DNSName }}`
which means, use the same template for both services `DNSService = `
and handle the name variable here:
```
coreDNSServiceBytes, err := kubeadmutil.ParseTemplate(CoreDNSService, struct{ DNSIP string }{
```